assets/main.js

```diff
@@ -1,36 +1,46 @@
+/*
+ * Libraries that should be required.
+ */
+var EDDBK_WIZARD_REQUIRE_LIBS = {
+  cjs: 'https://rawgit.com/guybedford/cjs/master/cjs',

```

Comment:
Can this map be stored in config somehow?

Reply:
Theoretically we can pass this URLs from backend. So it will be configurable.